blog.html

```diff
@@ -0,0 +1,64 @@
+
+
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog!</title>
+    <link href="https://fonts.googleapis.com/css?family=Courgette" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans:700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans:400" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Arbutus+Slab" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+
+      <header>
+        <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
+        <h1>Seattle's premier dog adoption agency</h1>
+        <img class="cart-img" src="images/cart-icon.svg" alt="Cart icon">
+        <a href="checkout.html" class="checkout-button">Checkout</a>
+      </header>
+
+      <nav>
+        <a href="index.html">Home&nbsp;&nbsp;/</a>
```

Comment:
Handling the / this way certainly works, but if you have time see if you can research how to use the after pseudo element to inject these with css. You'll want to use the content property. Totally not required, but would be a nice touch.